.containerignore

```diff
@@ -1 +1,2 @@
-rh-zync/zync/.git
\ No newline at end of file
+rh-zync/zync/.git
+rh-porta/porta/.git
```

Comment:
We have some `.dockerignore` files in porta and maybe zync. Maybe consider somehow using them dynamically.

Reply:
Good to know, thanks! I'll keep this in mind and try to bring those in, in the next round of changes. 👌🏼